src/trio/_tests/test_ssl.py

```diff
@@ -711,7 +711,7 @@ async def sleep_then_wait_writable() -> None:
 
     async def sleeper_with_slow_wait_writable_and_expect(method: str) -> None:
         if method == "wait_send_all_might_not_block":
-            await trio.sleep(100000)
+            await trio.sleep_forever()
         elif method == "expect":
             await trio.sleep(1000)
```

Comment:
surely both or neither of these should be changed, and not one of them.

Reply:
Fixed in new version